camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java

```diff
@@ -137,12 +139,13 @@ public void before() throws IOException, NoSuchFieldException, IllegalAccessExce
   public void after() throws IOException {
     // Delete all camus data
     folder.delete();
+    _mocks.clear();
```

Comment:
Still keeping _mocks as instance variable. Otherwise, a lot of methods need to add a parameter, which is cumbersome.